examples/hpc-slurm.yaml

```diff
@@ -28,53 +28,54 @@ vars:
 deployment_groups:
 - group: primary
   modules:
-  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
-  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
-  # Example - ./resources/network/vpc
-  - id: network1
+  # Source is an embedded module, denoted by "modules/*" without ./, ../, /
+  # as a prefix. To refer to a local module, prefix with ./, ../ or /
+  # Example - ./modules/network/vpc
+  - id: network
     source: modules/network/vpc
 
   - id: homefs
     source: modules/file-system/filestore
-    use: [network1]
+    use: [network]
     settings:
       local_mount: /home
 
-  - id: debug_node_group
-    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+  - id: debug_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
     settings:
       node_count_dynamic_max: 4
       machine_type: n2-standard-2
+      enable_placement: false # the default is: true
 
   - id: debug_partition
-    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    source: community/modules/compute/schedmd-slurm-gcp-v6-partition
     use:
-    - network1
     - homefs
-    - debug_node_group
+    - debug_nodeset
     settings:
       partition_name: debug
       exclusive: false # allows nodes to stay up after jobs are done
```

Comment:
I think it is important to keep placement disabled. In fact, the usual behavior is that if you enable placement, then exclusive is forced to true. We did have validation that warned the user about that.  So please make sure we have it again in V6.

Reply:
> I think it is important to keep placement disabled. 

We do, the setting moved from partition to nodeset ,  see L49

> We did have validation that warned the user about that. So please make sure we have it again in V6.

We do:

```hcl
  precondition {
    condition     = (length(local.use_placement) == 0) || var.exclusive
    error_message = "If any nodeset `enable_placement`, `var.exclusive` must be set true"
  }
```